src/main/java/de/rabea/request/RangeParser.java

```diff
@@ -0,0 +1,72 @@
+package de.rabea.request;
+
+import static java.util.Arrays.copyOfRange;
+
+public class RangeParser {
+
+    private final byte[] text;
+    private final String range;
+    private int textLength;
+    private int rangeCharacterLength;
+
+    public RangeParser(byte[] text, String rangeWithDescription) {
+        this.text = text;
+        this.range = strip(rangeWithDescription);
+        this.textLength = text.length;
+        this.rangeCharacterLength = range.length();
+    }
+
+    private String strip(String rangeWithDescription) {
+        return rangeWithDescription.substring(6);
```

Comment:
Not a big thing but maybe it's possible to give these magic numbers in here a name?
I assume `6` comes from `"range=".length()`, doesn't it?